attachment_edit/static/src/css/attachment_edit.css

```diff
@@ -0,0 +1,22 @@
+.openerp .oe_sidebar .oe_dropdown_menu li .oe-sidebar-attachment-edit
+{
+    position: absolute;
```

Comment:
unfortunately not, because the whole setup from core only allows absolute placements.
I needed the same thing for https://github.com/OCA/knowledge/blob/9.0/attachment_preview/static/src/css/attachment_preview.css, which is why those two modules can coexist in peace. But that doesn't scale at all or course.

What we need to do this right is in my opinion one module that moves core's delete button (the cross) into a qweb template, then all subsequent modules can simply add their buttons to this template. I'm quite sure to remember you, me and some other person had a similar discussion on some other PR, but I can't find it back now.

How about this: I hereby pledge that if you let the absolute placement slip here, I'll provide the base module described above and appropriate PRs for `attachment_edit` and `attachment_preview` as soon as a PR for a third button-to-attachment-adding module is PRed